.gitignore

```diff
@@ -2,3 +2,4 @@
 /target
 **/*.rs.bk
 Cargo.lock
+.DS_Store
```

Comment:
As @chrisduerr noted, this kind of files should be added to [a global gitignore on your computer](https://docs.github.com/en/free-pro-team@latest/github/using-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer) rather than in each project gitignore.